doc/contribute/index.rst

```diff
@@ -540,6 +540,13 @@ workflow here:
    By force pushing your update, your original pull request will be updated
    with your changes so you won't need to resubmit the pull request.
 
+   .. note:: While amending commits and force pushing is the recommended
+      model for adding reviewer-requested changes, it can cause
+      some unexpected behavior such as not being able to "View Changes"
+      to a GitHub PR (with GitHub complaining it can't find those commits).
+      Instead of viewing changes, you'll need to view the files in GitHub
+      as they exist after the forced push.
+
```

Comment:
Suggesting this:
> While amending commits and force pushing is a common review model outside github and the one recommended by Zephyr, it's not the main model supported by github and causes unexpected behavior such as not being able to use "View Changes" buttons except the last one (with GitHub complaining it can't find older commits) and not always being able to compare the latest reviewed version with the latest submitted version. When rewriting history github guarantees access to the latest version only.

Reply:
I took your suggested edit but broke up the long sentence...

Reply:
Long sentences? Me? Naaaaahh...